lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -27,6 +27,9 @@ describe 'StudentAdmin', ->
     it 'binds to the btn_rescore_entrance_exam on click event', =>
       expect(studentadmin.$btn_rescore_entrance_exam).toHandle 'click'
 
+    it 'binds to the $btn_skip_entrance_exam on click event', =>
```

Comment:
Seems like we'll need highlevel guidance regarding more coffeescript code. I've seen requests on other PRs to not extend our use of coffeescript. I've pinged @smagoun on this as this is probably we need to get some alignment on and get new example patterns to use.